tembo-pgmq-python/tembo_pgmq_python/async_queue.py

```diff
@@ -155,45 +155,79 @@ async def _list_queues_internal(self, conn):
         return queues
 
     @transaction
-    async def send(self, queue: str, message: dict, delay: int = 0, conn=None) -> int:
+    async def send(self, queue: str, message: dict, delay: int = 0, tz: str = None, conn=None) -> int:
         """Send a message to a queue."""
-        self.logger.debug(f"send called with queue='{queue}', message={message}, delay={delay}, conn={conn}")
+        self.logger.debug(f"send called with queue='{queue}', message={message}, delay={delay}, tz={tz}, conn={conn}")
```

Comment:
The python logging module is optimized for `%s`. Basically we don't need format and interpolate the log string unless we're actually going to log this message. So IMO, we should use `%s` on all calls to the python logging.

https://docs.python.org/3/howto/logging.html#optimization

Reply:
Also, I realize the codebase is already using fstring all over the place, so I'm not asking us to change this in this PR.

Reply:
@ChuckHend No issue then?

Reply:
Correct,  not an issue